tests/chart_tests/test_astronomer_houston_hook_job.py

```diff
@@ -112,3 +112,47 @@ def test_db_migration_job_custom_resources(self, kube_version):
         assert c_by_name["wait-for-db"]["resources"] == overrides
         assert c_by_name["houston-bootstrapper"]["resources"] == overrides
         assert c_by_name["houston-db-migrations-job"]["resources"] == overrides
+
+    def test_db_migration_job_init_containers_disabled_with_custom_secret_name(self, kube_version):
```

Comment:
We don't appear to have any tests that `airflowBackendSecretName` or `backendSecretName` function how we expect them to. We only have these tests for the initContainer.